schema/crdb/restrict-network-actions/up.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE omicron.public.silo
+    ADD COLUMN IF NOT EXISTS restrict_network_actions BOOL
+        NOT NULL
+        DEFAULT FALSE;
```

Comment:
I think you want a separate step here that removes the `DEFAULT`:
https://github.com/oxidecomputer/omicron/tree/a65cda6f7d1e969caa7fd72423e2ed8aa91cb386/schema/crdb#211-adding-a-new-column-without-a-default-value